### ISSUE TITLE
Improving the sourcing time ~100ms

### ISSUE THREAD

**Issue by PezCoder** [scmbreeze/scm_breeze#282]
#### What ?
```zsh
# Sourcing command in my ~/.zshrc
[ -s "/Users/pezcoder/.scm_breeze/scm_breeze.sh" ] && source "/Users/pezcoder/.scm_breeze/scm_breeze.sh"
```
- The first one's is sourcing without scm_breeze (~200ms)
& the later one's are with scm_breeze (~300ms)

![screenshot 2018-10-25 at 12 14 07 pm](https://user-images.githubusercontent.com/8057916/47480889-7e6eb280-d84f-11e8-93c3-dbf6b3ea7b00.png)

It takes around ~100 ms to source scm_breeze. Is there a way this can be improved further ?

**Comment by ghthor** [comment]
Yes, I agree. There has to absolutely be a way to speed this up.
